CRM/Tag/Form/Merge.php

```diff
@@ -88,8 +88,8 @@ public function postProcess() {
       CRM_Core_BAO_EntityTag::mergeTags($primary['id'], $tag['id']);
     }
 
-    if ($params['name'] != $primary['name']) {
-      civicrm_api3('Tag', 'create', ['id' => $primary['id'], 'name' => $params['name']]);
+    if ($params['label'] != $primary['label']) {
+      civicrm_api3('Tag', 'create', ['id' => $primary['id'], 'label' => $params['label']]);
     }
 
     $key = array_search($params['name'], $deleted);
```

Comment:
It needs it here and on line 84 too, otherwise the success message that comes up has names in it.